webapp/src/main/java/ar/edu/itba/paw/webapp/exceptions/MethodArgumentTypeMismatchExceptionMapper.java

```diff
@@ -0,0 +1,21 @@
+//package ar.edu.itba.paw.webapp.exceptions;
+//
+//import org.slf4j.Logger;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)